lib/schemas/interlock-constructor-input.js

```diff
@@ -0,0 +1,41 @@
+import * as path from "path";
```

Comment:
That's a good thought... I think I'm fine with either `lib/schemas/index.js` or `schemas/lib/index.js`, and then named exports corresponding to how they're used in the library module.  That sound good?